src/linear_algebra/std_basis.lean

```diff
@@ -0,0 +1,125 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl
+-/
+import linear_algebra.basic
+
+/-!
+# The standard basis
+
+This file defines the standard basis `std_basis R φ i b j`, which is `b` where `i = j` and `0`
+elsewhere.
+
+To give a concrete example, `std_basis R (λ (i : fin 3), R) i 1` gives the `i`th unit basis vector
+in `R³`, and `pi.is_basis_fun` proves this is a basis over `fin 3 → R`.
```

Comment:
Since this file doesn't actually prove that the standard basis is a basis, maybe the import hierarchy should go `basic.lean` -> `basis.lean` -> `std_basis.lean`?